index.bs

```diff
@@ -1527,6 +1546,48 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Preference enumeration (enum <dfn enum>AttestationPreference</dfn>) ### {#attestation}
```

Comment:
as noted above, am thinking "Attestation Presentation Preference" or "Attestation Conveyance Preference" is more accurately descriptive